SimCalorimetry/HcalTrigPrimAlgos/src/HcalTriggerPrimitiveAlgo.cc

```diff
@@ -254,9 +254,14 @@ void HcalTriggerPrimitiveAlgo::addSignal(const QIE10DataFrame& frame) {
     }
   } else if (detId.det() == DetId::Calo && detId.subdetId() == HcalZDCDetId::SubdetectorId) {
     HcalZDCDetId detId = frame.detid();
+    // skip RPD Channels
     if (detId.section() != HcalZDCDetId::EM && detId.section() != HcalZDCDetId::HAD) {
       return;
     }
+    // skip FSC and dummy channels
```

Comment:
Just for my understanding: "skip FSC **and** dummy channels" or rather "skip FSC dummy channels"

Reply:
This will skip both FSC channels as well as true ZDC dummy channels!